### DISCUSSION TITLE
Set dark theme even for login screen

### DISCUSSION THREAD

**Discussion by TheZoker** [kimai/kimai#4687] [General]
Hi there,

I noticed, that only after I log in, the theme switches to a dark theme. 
How can I set the application to have the dark theme by default and therefore also use it on the login screen?

Thanks!

**Comment by kevinpapst** [comment]
Interesting question, I never thought about it. I move it to the feature request section.

**Reply by kevinpapst** [reply]
Follow it here: https://github.com/kimai/kimai/issues/4688
